lib/utils/retry.go

```diff
@@ -33,6 +33,11 @@ import (
 // repeated calls.
 var HalfJitter = NewHalfJitter()
 
+// SeventhJitter is a global jitter instance used for one-off jitters.
+// Prefer instantiating a new jitter instance for operations that require
+// repeated calls.
+var SeventhJitter = NewSeventhJitter()
```

Comment:
Is this used anywhere?

Reply:
Currently just in `srv/heartbeatv2.go` currently, but seemed worth creating the global for it since we get a lot of use out of the corresponding `utils.HalfJitter`.